src/services/sessions.js

```diff
@@ -85,6 +85,14 @@ module.exports = class SessionsHelper {
 			const mentorIdToCheck = bodyData.mentor_id || loggedInUserId
 			const isSessionCreatedByManager = !!bodyData.mentor_id
 
+			if (bodyData.type == common.SESSION_TYPE.PRIVATE && menteeIdsToEnroll.length === 0) {
+				return responses.failureResponse({
+					message: 'MENTEES_NOT_FOUND',
```

Comment:
Is the message added in en.json?

Reply:
Yes , it is added , i am using the existing messages only .